plugin.go

```diff
@@ -0,0 +1,17 @@
+package root
```

Comment:
Naming is hard. I thought about using `manifest` but `manifest.Manifest` would stutter. Ideas?